src/main/java/seedu/address/model/tag/UniqueTagList.java

```diff
@@ -55,14 +53,9 @@ public boolean hasTag(Tag tag) {
         return false;
     }
 
-    public void setTags(UniqueTagList replacement) {
-        requireNonNull(replacement);
-        tagArrayList.setAll(replacement.tagArrayList);
-    }
-
     /**
-     * Replaces the contents of this list with {@code persons}.
-     * {@code persons} must not contain duplicate persons.
+     * Changes the old tags to a new list of tags
+     * @param tags The new list of tags.
```

Comment:
I believe that there should not be periods `.` at the end of `@param` javadocs.

Reply:
Ok. I made the changes.